__integrations__/docker-compose.yml

```diff
@@ -28,6 +28,14 @@ services:
       - 10000:10000
       - 10001:10001
       - 10002:10002
+  
+  durable-functions-storage-account:
+    image: mcr.microsoft.com/azure-storage/azurite
+    command: "azurite --loose --blobHost 0.0.0.0 --blobPort 10003 --queueHost 0.0.0.0 --queuePort 10004 --tableHost 0.0.0.0 --tablePort 10005"
+    ports:
+      - 10003:10003
+      - 10004:10004
+      - 10005:10005
```

Comment:
I think we may not need it anymore due to https://github.com/pagopa/io-functions-services/pull/169

Reply:
Done in [c7b5b63](https://github.com/pagopa/io-functions-services/pull/171/commits/c7b5b639fc68d175180f6dc0f555de16f0610652)